Updated TrustedHosts commands

SteveBurkettNZ opened the pull request:
Changed non-standard quote marks in command lines, added quote marks around file names in GetThroughput examples to maintain consistency, corrected typo of "Hyp-erV_ProfiledData" in GetThroughput command line.

azurecla commented:
Hi __@nickfloyd__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@nickfloyd : Thanks for your contribution to the Azure documentation! The author, @nickfloyd, has been notified to review your proposed change.

PRmerger commented:
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.

It�s easy to access the private repo version of your articles in GitHub. Just add �pr to the URL, as follows: 

Public repo URL: https://github.com/Azure/azure-content/blob/master/articles/azure-portal-overview.md

Private repo URL: https://github.com/Azure/azure-content-pr/blob/master/articles/azure-portal-overview.md

We've closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, see the contributor guide instructions.

cephalin commented:
@PRmerger He's not a Microsoft author. But he will take over as the main author of this article.

cephalin commented:
@nickfloyd I think @PRmerger isn't seeing this because the PR is already closed. I can't reopen the PR. Can you reopen it?

nickfloyd commented:
@cephalin I cannot.  I could always resubmit and sidekick you on the new PR? LMK.

cephalin commented:
@nickfloyd Sounds good. Go for it.

nickfloyd commented:
@cephalin - done - see related PR [here](https://github.com/Azure/azure-content/pull/7437)

SteveBurkettNZ committed:
Corrected further non-standard quote marks + typo

Changed non-standard quote marks in command lines, added quote marks around file names in GenerateReports examples to maintain consistency, corrected typo of "Hyp-erV_ProfiledData" in GetThroughput command line.
